.gitignore

```diff
@@ -22,6 +22,7 @@ yarn-error.log
 target
 
 packages/template-set/.env
+packages/prepackaged-site/bin
```

Comment:
out of curiosity, what does generate this `bin/` folder?

Reply:
Probably the vscode java/maven plugin, but I have no idea